tests/test_prompt_template_create_prompt.py

```diff
@@ -0,0 +1,196 @@
+
+@SSTbrostrom: Jeg forstår godt mange synes nye krav om mundbind er vidtgående. Skal også selv vænne mig til nye krav skal på i butik osv. men det er altså ikke slemt. Dokumentationen har vi, og med aktuelle smittetryk i Danmark er det rettidig omhu #sundpol #SundhedForAlle  https://t.co/SlmGZtN5dh
+<subject-1>@SSTbrostrom</subject-1>: Jeg <predicate-1>forstår </predicate-1>godt <object-1><subject-2>mange </subject-2><predicate-2>synes </predicate-2><object-2><subject-3>nye krav om mundbind </subject-3><predicate-3>er </predicate-3><object-3>vidtgående</object-1></object-2></object-3>. Skal også selv vænne mig til nye krav skal på i butik osv. men <subject-4>det </subject-4><predicate-4>er </predicate-4>altså <object-4>ikke slemt</object-4>. <object-5>Dokumentationen </object-5><predicate-5>har </predicate-5><subject-5>vi</subject-5>, og med aktuelle smittetryk i Danmark <predicate-6>er </predicate-6><subject-6>det </subject-6><object-6>rettidig omhu </object-6>#sundpol #SundhedForAlle  https://t.co/SlmGZtN5dh
+
+@Ihavequestione: @SSTbrostrom Men vi bragte da smitten ned i foråret uden mundbind, så det er underligt det nu skal indføres 🤷🏻\u200d♂️
+@Ihavequestione: @SSTbrostrom Men <subject-1>vi </subject-1><predicate-1>bragte </predicate-1>da <object-1>smitten ned i foråret </object-1>uden mundbind, så <subject-2>det </subject-2><predicate-2>er </predicate-2><object-2>underligt <subject-3>det </subject-3><object-3>nu </object-3><predicate-3>skal indføres </object-2></predicate-3>🤷🏻\u200d♂️
+
+@siggithilde: @Ihavequestione @svogdrup @GianniRivera69 @SSTbrostrom Det første punkt under potentielle fordele er alt rigeligt til at indføre mundbindstvang overalt i det offentlige rum.
+@siggithilde: @Ihavequestione @svogdrup @GianniRivera69 @SSTbrostrom <subject-1>Det første punkt under potentielle fordele </subject-1><predicate-1>er </predicate-1>alt <object-1>rigeligt til at indføre mundbindstvang overalt i det offentlige rum</object-1>.
+
+@user1: This is a test tweet, I am commenting on something someone else said. @user2 this is not good enough.
+"""  # noqa: E501
+
+
+examples, task_description, test_tweet = get_examples_task_introduction()
+
+
+@pytest.mark.parametrize(
+    "template, target, examples, task_description",
+    [
+        (PromptTemplate1, test_tweet, examples, task_description),
+        (PromptTemplate2, test_tweet, examples, task_description),
+        (MarkdownPromptTemplate1, test_tweet, examples, task_description),
+        (MarkdownPromptTemplate2, test_tweet, examples, task_description),
+        (XMLStylePromptTemplate, test_tweet, examples, task_description),
+    ],
+)
+def test_PromptTemplate_create_prompt(template, target, examples, task_description):
+    template = template(task_description, examples)
+    prompt = template.create_prompt(target)
+    expected_prompt = get_expected_prompt(template)
```

Comment:
Hmm this fetch approach does not allow for multiple tests of the same template. Is there any reason for using that rather than just passing it in as an argument?